galasa-parent/dev.galasa.framework.api.users/src/main/java/dev/galasa/framework/api/users/internal/routes/UsersRoute.java

```diff
@@ -38,11 +40,14 @@ public class UsersRoute extends BaseRoute {
 
     private IFramework framework;
     private Environment env;
+    private IAuthStoreService authStoreService;
 
-    public UsersRoute(ResponseBuilder responseBuilder, IFramework framework, Environment env) {
+    public UsersRoute(ResponseBuilder responseBuilder, IFramework framework, Environment env,
```

Comment:
It doesn't look like `framework` is being used anywhere here, you might be able to remove it